DigitalLearningSolutions.Web/Scripts/trackingSystem/emailDelegates.ts

```diff
@@ -0,0 +1,26 @@
+import { SearchSortFilterAndPaginate } from '../searchSortFilterAndPaginate/searchSortFilterAndPaginate';
+
+const selectedElements = document.querySelectorAll('.delegate-checkbox:checked') as NodeListOf<HTMLInputElement>;
+const selectedIds = Array.from(selectedElements).map((el) => el.value);
+const queryString = '?selectedIds='.concat(selectedIds.join('&selectedIds='));
+const route = `TrackingSystem/Delegates/Email/AllEmailDelegateItems${queryString}`;
+
+// eslint-disable-next-line no-new
+new SearchSortFilterAndPaginate(route, false, false, true, 'EmailDelegateFilter');
+
+function selectAll(): void {
+  const allCheckboxes = document.querySelectorAll('.delegate-checkbox') as NodeListOf<HTMLInputElement>;
+  allCheckboxes.forEach((checkbox) => { if (!checkbox.checked) checkbox.click(); });

```

Comment:
Super minor, but might be neater to put the if statement on a new line - ditto below.